src/Console/ConfigurationResolver.php

```diff
@@ -253,7 +255,21 @@ public function getConfigFile()
     public function getDiffer()
     {
         if (null === $this->differ) {
-            $this->differ = false === $this->options['diff'] ? new NullDiffer() : new SebastianBergmannDiffer();
+            if (false !== $this->options['differ']) {
+                // 'diff' value does not matter here
+                if ('sbd' === $this->options['differ']) {
```

Comment:
you won't _always_ get the `full` file diff even when using the `long` format. 
You get an extended diff in Unified Format (well, on 1.14.1 I hope, before that the format is a bit more loose, i.e. the diff is in a format that can be described as the `sbd` diff. format)